### ISSUE TITLE
[Bug]: `EasyPostTimeInTransitData` inaccessible

### ISSUE THREAD

**Issue by zarubto4** [EasyPost/easypost-java#304]
### Software Version

7.1.0

### Language Version

Java 17

### Operating System

MacOS

### What happened?

Hi, 

in object EasyPostTimeInTransitData here 
 https://github.com/EasyPost/easypost-java/blame/master/src/main/java/com/easypost/model/EstimatedDeliveryDate.java 
you have all values as private without abstract parent services. 

So it's not possible to use that in Java. 

![Snímek obrazovky 2024-03-21 v 2 20 24](https://github.com/EasyPost/easypost-java/assets/16296782/16e72f6f-879e-4df9-92fb-8e8715110331)

### What was expected?

Set is as public or create abstract class with `@Getter` 

### Sample Code

```java
public String getPlannedShipDate() {
        return plannedShipDate;
    }

 public String getEasyPostEstimatedDeliveryDate() {
        return easypostEstimatedDeliveryDate;
    }
```

**Comment by nwithan8** [comment]
@zarubto4 Thank you for bringing this to our attention. It appears the class was not explicitly marked as public, making it inaccessible as you've pointed out. This is being rectified in #305 and will be included in our next release.

**Comment by jchen293** [comment]
@zarubto4 We have just released v7.1.1 for this patch, thanks for submitting this issue!
